products/react.md

```diff
@@ -0,0 +1,35 @@
+---
+title: React
+permalink: /react
+iconSlug: react
+layout: post
+category: framework
+sortReleasesBy: 'releaseCycle'
+releasePolicyLink: https://reactjs.org/docs/release-channels.html
+changelogTemplate: https://github.com/facebook/react/releases/tag/v__LATEST__
+activeSupportColumn: true
+releaseDateColumn: true
+auto:
+  git: https://github.com/react/react.git
+releases:
+  - releaseCycle: "18"
+    release: 2022-03-29
+    support: true
+    eol: false
+    latest: "18.0.0"
+  - releaseCycle: "17"
+    lts: false
+    release: 2020-10-20
+    eol: 2021-03-22
```

Comment:
Shouldn't this be `2022-03-29` (ie, same as v18 release?)

Reply:
@JasonMongue Can you check this please?

Reply:
> Shouldn't this be `2022-03-29` (ie, same as v18 release?)

[Yes](https://github.com/reactjs/reactjs.org/issues/1745#issuecomment-466767389)
Made the fix: https://github.com/endoflife-date/endoflife.date/pull/1506